### PR TITLE
Adds form for selection of Pokemons by the opponent

### PR DESCRIPTION
**Pull request by jaenia**
[As the opponent Trainer I select my PKNs to battle](https://trello.com/c/oXVCG30U/4-as-the-opponent-trainer-i-select-my-pkns-to-battle)

### Description
This PR adds the form for the opponent select Pokemons to battle.

### How to test
- [ ] Click to add a battle
- [ ] Select your opponent
- [ ] Choose creator's Pokemons and create the battle. You'll be redirected to the form for opponent's Pokemons.
- [ ] Choose opponent's Pokemons and click on "Battle!" button
- [ ] Make sure that the battle now has creator's and opponent's Pokemon teams

### CODE REVIEW

File: backend/battles/views.py [hunk 1]
```diff
@@ -14,6 +15,15 @@ def get_form_kwargs(self):
         kwargs["current_user"] = User.objects.first()
         return kwargs
 
+    def get_success_url(self):
+        return reverse("battles:battle_update_opponent_pokemons", kwargs={"pk": self.object.pk})
+
+
+class BattleUpdateOpponentPokemons(UpdateView):
```
Comment (filipeximenes) [inline backend/battles/views.py:22]: Doesn't this need a `get_success_url`?

Comment (jaenia) [reply backend/battles/views.py:22]: Well, the default behavior for `UpdateView` seems to be the redirection for the `DetailView`. And this is correct for my current context. So, everything looks fine for now. What do you think about it?

Comment (filipeximenes) [reply backend/battles/views.py:22]: Aha! 
```
You don’t even need to provide a success_url for CreateView or UpdateView - they will use get_absolute_url() on the model object if available.
```
https://docs.djangoproject.com/en/3.1/topics/class-based-views/generic-editing/#model-forms

Cool!

**Review by filipeximenes — APPROVED** [review]

File: backend/battles/views.py (hunk 1 above)
Comment (jaenia) [reply backend/battles/views.py:22]: Yes, that's it! I was looking for the documention too, but I didn't managed to find it.

**Pull request merged**